app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">EasySurvey</string>
+    <string name="app_description">Making surveys easy.</string>
```

Comment:
I added this string to add to the splash screen title. This adds a bit more to the screen instead of just having the title of the app.